src/test/unit/process-profile.test.js

```diff
@@ -247,44 +255,167 @@ describe('serializeProfile', function() {
     );
     expect(roundtrip).toEqual(secondRountrip);
   });
+});
 
-  describe('removing network urls', function() {
-    it('should remove all pages information', async function() {
-      const profile = processProfile(createGeckoProfile());
+describe('sanitizePII', function() {
+  it('should sanitize the threads if they are provided', function() {
+    const profile = processProfile(createGeckoProfile());
+    // There are 3 threads in the beginning.
+    expect(profile.threads.length).toEqual(3);
+    const PIIToRemove: RemoveProfileInformation = {
```

Comment:
It might be more obvious to the reader to do this as a function like:

`const piiToRemove = getRemoveProfileInformation({ shouldRemoveThreads: [0, 2] })`